src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -23,24 +23,63 @@
 
     <section name="What is Javadoc comment">
       <p>
-      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition or field definition.
+        Javadoc comment is a multiline comment <code>/*  */</code> that starts with the <b>*</b> character and placed above the class definition, interface definition, enum definition, method definition or field definition.
+        If an annotation precedes any of the definitions listed above, then the comment should be placed before the annotation.
+        If a number of comments are put sequentially, only the last one will be used.
```

Comment:
1)
>comments

"multiline comments with javadoc identifiers". 
All of them are comments, looks like javadoc ..... but we call javadoc comment only one that most matching our criteria.

2) 
>only the last one will be used

"last" is not good word here, as in your example last is `/** Not a javadoc (ignored). */` , fact that it is separated by new line is not countable.
Lets use "most nearest on top(above) multiline comment with javadoc identifier".